ifttt/helpers.py

```diff
@@ -1,6 +1,13 @@
 import json
+from django.conf import settings
+from datetime import datetime
 
-class IfThisThenThatHelpers():
+def convert_string_time(str):
```

Comment:
Moved this outside the class. Now you can use it just like a function (e.g.: `convert_string_time`), or import it with: `from helpers import convert_string_time `